trading_calendars/exchange_calendar_xbue.py

```diff
@@ -184,7 +184,7 @@ def not_2018(holidays):
     'Bank Holiday',
     month=11,
     day=6,
-    observance=not_2018,
+    end_date='2017',
```

Comment:
Sorry that this isn't clear, but `end_date='2017'` is equivalent to `end_date='01-01-2017'`, which means this would not be a holiday in 2017, it would last be a holiday in 2016. Is that what you intended?

Reply:
nope, that was a mistake, nice catch.